repository/g.json

```diff
@@ -1627,6 +1627,19 @@
 			]
 		},
 		{
+  		"name": "Green Writing Color Scheme",
+        	"details": "https://github.com/caoyeacy/sublime-writing-color-scheme-green",
+                "description": "Green Writing Color Scheme For Writing LaTeX",
+                "labels": ["color"],
+  		"releases": [
```

Comment:
The indenting on this line and the four above is incorrect. They should be indented by one tab beyond the `{`.